methods/add_teacher.rb

```diff
@@ -0,0 +1,12 @@
+def add_teacher
+  print 'Enter the name of the teacher: '
+  name = gets.chomp.to_s
+  print 'Enter the age of the teacher: '
+  age = gets.chomp.to_i
+  print "And teacher's specialization: "
```

Comment:
- [ ] Following the principle of single responsibility, please create a method whose sole responsibility is to request and validate teacher input